### DISCUSSION TITLE
Where UD comes from?

### DISCUSSION THREAD

**Discussion by kelvindesman** [fastapi-users/fastapi-users#1429] [Q&A]
https://github.com/fastapi-users/fastapi-users-db-mongodb/blob/b4208069e75ec094b95807f999de46685b2d0495/fastapi_users_db_mongodb/__init__.py#L5

**Comment by frankie567** [comment]
This database adapter is not supported anymore. You should use Beanie: https://fastapi-users.github.io/fastapi-users/latest/configuration/databases/beanie/
